jarbas/core/tests/test_reimbursement_model.py

```diff
@@ -89,6 +89,29 @@ def test_suspicions(self):
         Reimbursement.objects.create(**self.data)
         self.assertEqual(1, Reimbursement.objects.suspicions().count())
 
+    def test_has_receipt_url(self):
```

Comment:
Do you mean this tests @cuducos?

Reply:
He is talking about the line 65 of readme, @matheushf. The param there is "has_receipt", but on view you call for "has_receipt_url" (line 34)